observations/models.py

```diff
@@ -76,21 +77,25 @@ class Observation(PolymorphicModel):
     """
     value = models.ForeignKey(
         AllowedValue, blank=False, null=True,
+        on_delete=models.CASCADE,
         related_name='instances')
     time = models.DateTimeField(
         db_index=True,
         help_text='Exact time the observation was made')
     unit = models.ForeignKey(
         services_models.Unit, blank=False, null=False,
+        on_delete=models.CASCADE,
         help_text='The unit the observation is about',
         related_name='observation_history')
     units = models.ManyToManyField(
         services_models.Unit, through='UnitLatestObservation')
     auth = models.ForeignKey(
-        'PluralityAuthToken', null=True)
+        'PluralityAuthToken', null=True,
+        on_delete=models.CASCADE)
     property = models.ForeignKey(
         ObservableProperty,
         blank=False, null=False,
+        on_delete=models.CASCADE,
```

Comment:
Should we prevent deleting observable properties if somebody has made observations already? Soft delete perhaps?

Reply:
Observation deletion prevented in 35306b441fac9c4352c57a3873c114a2e1912663
Actual soft deletion added to backlog.